beacon_node/network/src/sync/mod.rs

```diff
@@ -2,7 +2,7 @@
 //!
 //! Stores the various syncing methods for the beacon chain.
 mod backfill_sync;
-mod block_lookups;
+pub mod block_lookups;
```

Comment:
this is now exposed because `spawn_delayed_lookup_service` is moved to router, but can be reverted if we decide to have the `spawn_delayed_lookup_service` in sync manager.